hooks/tk-3dsmaxplus_actions.py

```diff
@@ -169,14 +169,14 @@ def _import_alembic(self, path):
             )
         )
     
-    def _merge(self, path, sg_publish_data):
+    def _merge(self, path_utf8, sg_publish_data):
```

Comment:
nice and clear!

Reply:
questions:

- why are we not doing this for alembic?
- to avoid repetition could we do the conversion in `execute_action` above (after line 135)?
- if this is not the case, we need some clear comments explaining this, to help future readers and maintainers of the code. Right now it's not quite clear why the code is laid out the way it is.